server/accounts.go

```diff
@@ -3148,7 +3148,9 @@ func (a *Account) expiredTimeout() {
 	// Collect the clients and expire them.
 	cs := a.getClients()
 	for _, c := range cs {
-		c.accountAuthExpired()
+		if !isInternalClient(c.kind) {
```

Comment:
@derekcollison I am not sure if this should be limited to JETSTREAM only or to all internal clients, or if this is even the right approach...

Reply:
This probably is the right approach, as I think NRG nodes run with `SYSTEM` client kinds.

Reply:
Agree should be fine. @neilalexander do we keep type even if in user account?

Reply:
Yes, it's kind `SYSTEM` either way.